test/jdk/sanity/client/lib/SwingSet3/src/com/sun/swingset3/demos/table/resources/TableDemo.properties

```diff
@@ -9,9 +9,9 @@ TableDemo.searchLabel=Search Titles and Recipients
 TableDemo.winnersLabel=Show Only Winners
 
 TableDemo.noDataStatusLabel=No data loaded
-TableDemo.loadingStatusLabel=Loading data: 
-TableDemo.rowCountLabel=Showing 
-TableDemo.searchCountLabel=Search found 
+TableDemo.loadingStatusLabel=Loading data:\u0020
+TableDemo.rowCountLabel=Showing\u0020
+TableDemo.searchCountLabel=Search found\u0020
```

Comment:
These are concatenated, so either keep the space or update the code to include the space.